### DISCUSSION TITLE
How to enable the DICOM-RT plug-in

### DISCUSSION THREAD

**Discussion by yidu-bjcancer** [nroduit/Weasis#365] [Q&A] [answered]
A demo picture of DICOM-RT(radiotherapy) is shown on the website of Weasis. But does anybody know how to use or enable the DICOM-RT plugin? I could find any clue in the Win10 desktop version (Weasis-4.0.3-x86-64.msi)

Thanks in advance

**Comment by nroduit** [comment, marked as answer]
For now, this plugin is not activated by default.

You have to change the configuration of the INSTALL_PATH/Weasis/???/app/conf/config.properties file and activate the 115 plugin (by uncommenting the 2 lines).
![image](https://user-images.githubusercontent.com/993975/210502442-140215bd-c49f-4092-b396-0d6f364aa2c6.png)

Then the RT Tool appears on the right. When a CT is linked to RT data, you just have to click on the "Load RT" button.
![image](https://user-images.githubusercontent.com/993975/210502458-94048a6d-2d45-42e1-a69d-bbc0d8ef2e92.png)

**Comment by nroduit** [comment]
It will be activated by default in the next release, see https://github.com/nroduit/Weasis/issues/395
